src/components/settingpage/leftcomponent/content.tsx

```diff
@@ -72,11 +72,11 @@ const Content: React.FC<IUserInfo> = ({ address, ethAddress, account }) => {
         <div className={title}>{t('settingpage.language')}</div>
         <button
           type="button"
-          onClick={() =>
-            i18n.language === 'en-US'
-              ? i18n.changeLanguage('zh-TW')
-              : i18n.changeLanguage('en-US')
-          }
+          onClick={() => {
+            if (i18n.language === 'en-US') i18n.changeLanguage('zh-TW');
+            else i18n.changeLanguage('en-US');
```

Comment:
Please user original `conditional operator` here.

Reply:
done

Reply:
You can use this to avoid lint error:
```tsx
i18n.changeLanguage(i18n.language === 'en-US' ? 'zh-TW' : 'en-US');
```